Zend/tests/nullsafe_operator/016.phpt

```diff
@@ -0,0 +1,35 @@
+--TEST--
```

Comment:
We should actually also test that `?->` in inner expressions by reference are ok.

```php
takes_ref(returns_ref($foo?->bar));
```